src/contexts/wizard/WizardDataProvider.tsx

```diff
@@ -43,15 +41,11 @@ type Props = { id: string; children: ReactNode };
 export default memo(function WizardDataProvider({ id, children }: Props) {
   const [queryRef, loadQuery] = useQueryLoader<OrganizeCollectionQuery>(organizeCollectionQuery);
 
-  const [isRefreshingNfts, setIsRefreshingNfts] = useState(false);
```

Comment:
this doesn't need to exist anymore

Reply:
@tbezman  we want to bring this back for the onboarding flow. 

user onboards
syncs nfts
gets to collection editor
still syncing, we don't want them to sync again